### PR TITLE
[4375] Missing Product Filter Placeholder Fix

### LINKED ISSUE #4375: Placeholder for Layered navigation is missing on PLP when opening page for the first time
**Description**:
Placeholder for Layered navigation is missing on PLP when opening page for the first time

**Expected behavior**:
Placeholder with loader is displayed 

**To Reproduce**:
1. Open PLP in incognito mode

**Actual result**:
Placeholder for Layered navigation is missing when opening page for the first time
https://drive.google.com/file/d/1imeCQR8J5znV1ij6djmpPv5--wQXBNuj/view?usp=sharing

### PR DESCRIPTION
**Pull request by leonazava**
**Related issue(s):**
* Fixes #4375

**Problem:**
* The placeholder for the product filter wasn't visible on a first load

**In this PR:**
* Fixed the issue with one CSS line to give the placeholder a visible background

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
1. Placeholders is good
2. 'Shopping Options' wasn't added
**Actual result:**
Screenshot 2022-03-24 at 08 15 59
**Expected result:**
Add 'Shopping Options' text and it should align with placeholder for category name
Screenshot 2022-03-24 at 08 27 38

**Comment by aleksandrakorolova** [comment]
2. 'Shopping Options' was added but it's not aligned with placeholder for category name (on the left side of screenshot - when user is opening PLP in the first time, the right side - the second time). Please, align 'Shopping Options' text with placeholder for category name as on the right side of screenshot)
Screenshot 2022-03-25 at 08 37 27

File: packages/scandipwa/src/component/CategoryFilterOverlay/CategoryFilterOverlay.container.js [hunk 1]
```diff
@@ -60,6 +61,7 @@ export class CategoryFilterOverlayContainer extends PureComponent {
         isProductsLoading: PropTypes.bool.isRequired,
         isSearchPage: PropTypes.bool.isRequired,
         totalPages: PropTypes.number.isRequired
+
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CategoryFilterOverlay/CategoryFilterOverlay.container.js:64]: Remove whitespace

File: packages/scandipwa/src/component/ProductConfigurableAttributes/ProductConfigurableAttributes.component.js [hunk 2]
```diff
@@ -27,7 +23,8 @@ import './ProductConfigurableAttributes.style';
 /** @namespace Component/ProductConfigurableAttributes/Component */
 export class ProductConfigurableAttributes extends PureComponent {
     static propTypes = {
-        numberOfPlaceholders: PropTypes.arrayOf(PropTypes.number),
+        // eslint-disable-next-line react/require-default-props
+        renderPlaceholder: PropTypes.func,
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/ProductConfigurableAttributes/ProductConfigurableAttributes.component.js:27]: Remove disabling eslint

File: packages/scandipwa/src/component/ProductConfigurableAttributes/ProductConfigurableAttributes.container.js [hunk 3]
```diff
@@ -26,6 +26,8 @@ import ProductConfigurableAttributes from './ProductConfigurableAttributes.compo
 /** @namespace Component/ProductConfigurableAttributes/Container */
 export class ProductConfigurableAttributesContainer extends PureComponent {
     static propTypes = {
+        // eslint-disable-next-line react/require-default-props
+        renderPlaceholder: PropTypes.func,
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/ProductConfigurableAttributes/ProductConfigurableAttributes.container.js:30]: Same here

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js [hunk 4]
```diff
@@ -207,22 +210,44 @@ export class CategoryPage extends PureComponent {
...
-            </p>
+            <>
+                { BIG_PLACEHOLDER_CONFIG.map((length, i) => (
+                    // eslint-disable-next-line react/no-array-index-key
+                    <div key={ i } block={ block } elem="SwatchList">
+                        { Array.from({ length }, (_, i) => (
+                            // eslint-disable-next-line react/no-array-index-key
+                            <div key={ i } block={ block } elem="Placeholder" />
+                        )) }
+                    </div>
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js:223]: I assume you can refactore this to avoid eslint disabling.

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**